autotst/calculator/gaussianTest.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+
+import os
+import itertools
+import logging
+import numpy as np
+from cclib.io import ccread
+
+import rmgpy
+from rmgpy.molecule import Molecule as RMGMolecule
+from rmgpy.reaction import Reaction as RMGReaction
+
+import autotst
+from autotst.reaction import Reaction, TS
+from autotst.species import Species, Conformer
+from autotst.geometry import Torsion
+from autotst.calculator.gaussian import Gaussian
+
+from cclib.io import ccread
+
+from ase import Atom, Atoms
+from ase.io.gaussian import read_gaussian, read_gaussian_out
+from ase.calculators.gaussian import Gaussian as ASEGaussian
+
+from hotbit import Hotbit
+class test_gaussian(unittest.TestCase):
+    def setUp(self):
+        rxn = Reaction(label='C+[O]O_[CH3]+OO')
+        rxn.generate_conformers(ase_calculator=Hotbit())
```

Comment:
Probably shouldn't be here until we get Hotbit resolved or we use a different calculator already included in ASE